realm/realm-library/src/main/java/io/realm/RealmList.java

```diff
@@ -381,6 +381,34 @@ public int size() {
         }
     }
 
+    /**
+     * Returns true if the list contains the specified element, when attached to a Realm. This
+     * method will query the native Realm core engine to quickly find the specified element.
+     *
+     * If this list is not attached to a Realm, the default {@link List#contains(Object)}
+     * operation will occur.
+     *
+     * @param object The element whose presence in this list is to be tested.
```

Comment:
The description should start with lowercase letter. `The element` -> `the element`